### ISSUE TITLE
Repo update?

### ISSUE THREAD

**Issue by PaulCombal** [PistonMiner/TF2Classic#13]
Hello TF2Classic team,
You are advertising this project as being open source on your official website, however, when someone complains that the code is not compiling, all you get to answer is "sorry, but this code is outdated and no longer used, issue closed". I believe this is not how open source is supposed to work, as indeed this repository has not been updated in half a year now, and is not marked as "no longer maintained" whatsoever in the Readme files.
It has also been impossible for me to guess whether or not the project is being worked on as your website doesn't include detailed update history, or any date at all.

I would like to know if you currently are still working on the game, even in closed source, if you moved the repo location, or if you intended to make the source code of this project available to everyone any time soon.

Thanks for your attention

**Comment by PistonMiner** [comment]
You are right, our website is outdated. Thank you for pointing that out, it'll be taken care of once we have the capacity to do so. And yes, the project is still being worked on, as is visible in the multitude of places for discussion of the project.

**Comment by PaulCombal** [comment]
Thanks for the quick reply!
I wish you luck for this project, also please leave a note somewhere explaining all of this to avoid getting confused persons like me!
Thanks again, and take care :)
